packages/build/tests/plugins/fixtures/plugins_cache_outdated/.netlify/plugins/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "netlify-local-plugins",
+  "description": "This directory contains Build plugins that have been automatically installed by Netlify.",
+  "version": "1.0.0",
+  "private": true,
+  "author": "Netlify",
+  "license": "MIT",
+  "dependencies": {
+    "netlify-plugin-contextual-env": "0.2.0"
```

Comment:
Outdated version. Latest in the tests is set to always be `0.3.0`. I.e. a new install should be done.